src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const tabSelected = tabs.find(tab => tab.id === activeTabId) || tabs[0];
+  const findActiveTab = tabs.find(item => {
+    return item.id === activeTabId;
+  });
+  const ifIsActive = item => {
```

Comment:
So, the main purpose of this function is to select a new tab, not to check if it is active. The name should be according 
```suggestion
  const handleTabSelect = item => {
```